src/explore-education-statistics-admin/src/pages/publication/__tests__/PublicationDetailsPage.test.tsx

```diff
@@ -0,0 +1,408 @@
+    },
+  };
+
+  const testThemes: Theme[] = [
+    {
+      id: 'theme-1',
+      slug: 'theme-1-slug',
+      summary: '',
+      title: 'Theme 1',
+      topics: [
+        {
+          id: 'theme-1-topic-1',
+          slug: 'theme-1-topic-1-slug',
+          themeId: 'theme-1',
+          title: 'Theme 1 Topic 1',
+        },
+        {
+          id: 'theme-1-topic-2',
+          slug: 'theme-1-topic-2-slug',
+          themeId: 'theme-1',
+          title: 'Theme 1 Topic 2',
+        },
+      ],
+    },
+    {
+      id: 'theme-2',
+      slug: 'theme-2-slug',
+      summary: '',
+      title: 'Theme 2',
+      topics: [
+        {
+          id: 'theme-2-topic-1',
+          slug: 'theme-2-topic-1-slug',
+          themeId: 'theme-2',
+          title: 'Theme 2 Topic 1',
+        },
+        {
+          id: 'theme-2-topic-2',
+          slug: 'theme-2-topic-2-slug',
+          themeId: 'theme-2',
+          title: 'Theme 2 Topic 2',
+        },
+      ],
+    },
+  ];
+
+  const testPublicationSummaries: PublicationSummary[] = [
+    {
+      id: 'publication-1',
+      slug: 'publication-1-slug',
+      title: 'Publication 1',
+    },
+    {
+      id: 'publication-2',
+      slug: 'publication-2-slug',
+      title: 'Publication 2',
+    },
+    {
+      id: 'publication-3',
+      slug: 'publication-3-slug',
+      title: 'Publication 3',
+    },
+  ];
+
+  beforeEach(() => {
+    themeService.getTheme.mockResolvedValue(testThemes[0]);
+    themeService.getThemes.mockResolvedValue(testThemes);
+    publicationService.getPublication.mockResolvedValue(
+      testPublicationSummaries[1] as BasicPublicationDetails,
+    );
+    publicationService.getPublicationSummaries.mockResolvedValue(
+      testPublicationSummaries,
+    );
+  });
+
+  test('renders the read only page correctly', async () => {
+    renderPage(testPublication);
+
+    await waitFor(() => {
+      expect(screen.getByText('Publication details')).toBeInTheDocument();
+    });
+
+    expect(screen.getByTestId('Publication title')).toHaveTextContent(
+      'Publication 1',
+    );
+    expect(screen.getByTestId('Theme')).toHaveTextContent('Theme 1');
+    expect(screen.getByTestId('Topic')).toHaveTextContent('Theme 1 Topic 2');
+    expect(screen.getByTestId('Superseding publication')).toHaveTextContent(
+      'This publication is not archived',
+    );
+    expect(
+      screen.getByRole('button', { name: 'Edit publication details' }),
+    ).toBeInTheDocument();
+  });
+
+  test('shows the superseding publication if the publication is archived', async () => {
+    renderPage({ ...testPublication, supersededById: 'publication-2' });
+
+    await waitFor(() => {
+      expect(screen.getByText('Publication details')).toBeInTheDocument();
+    });
+
+    expect(screen.getByTestId('Superseding publication')).toHaveTextContent(
+      'Publication 2',
+    );
+  });
+
+  test('shows the form when the edit button is clicked', async () => {
```

Comment:
Just for better organisation, might be nice if we grouped all the form related tests into their own `describe` block